specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.4/TextAnalytics.json

```diff
@@ -2319,6 +2325,25 @@
entAgencyNumber", "EEPersonalIdentificationCode", "EUDebitCardNumber", "EUDriversLicenseNumber", "EUGPSCoordinates", "EUNationalIdentificationNumber", "EUPassportNumber", "EUSocialSecurityNumber", "EUTaxIdentificationNumber", "FIEuropeanHealthNumber", "FINationalID", "FINationalIDV2", "FIPassportNumber", "FRDriversLicenseNumber", "FRHealthInsuranceNumber", "FRNationalID", "FRPassportNumber", "FRSocialSecurityNumber", "FRTaxIdentificationNumber", "FRValueAddedTaxNumber", "DEDriversLicenseNumber", "DEPassportNumber", "DEIdentityCardNumber", "DETaxIdentificationNumber", "DEValueAddedNumber", "GRNationalIDCard", "GRNationalIDV2", "GRTaxidentificationNumber", "HKIdentityCardNumber", "HUValueAddedNumber", "HUPersonalIdentificationNumber", "HUTaxidentificationNumber", "INPermanentAccount", "INUniqueIdentificationNumber", "IDIdentityCardNumber", "InternationlBankingAccountNumber", "IEPersonalPublicServiceNumber", "IEPersonalPublicServiceNumberV2", "ILBankAccountNumber", "ILNationalID", "ITDriversLicenseNumber", "ITFiscalCode", "ITValueAddedTaxNumber", "JPBankAccountNumber", "JPDriversLicenseNumber", "JPPassportNumber", "JPResidentRegistrationNumber", "JPSocialInsuranceNumber", "JPMyNumberCorporate", "JPMyNumberPersonal", "JPResidenceCardNumber", "LVPersonalCode", "LTPersonalCode", "LUNationalIdentificationNumberNatural", "LUNationalIdentificationNumberNonNatural", "MYIdentityCardNumber", "MAIdentityCardNumber", "MTTaxIDNumber", "NLCitizensServiceNumber", "NLCitizensServiceNumberV2", "NLTaxIdentificationNumber", "NLValueAddedTaxNumber", "NZBankAccountNumber", "NZDriversLicenseNumber", "NZInlandRevenueNumber", "NZMinistryOfHealthNumber", "NZSocialWelfareNumber", "NOIdentityNumber", "PHUnifiedMultiPurposeIDNumber", "PLIdentityCard", "PLNationalID", "PLNationalIDV2", "PLPassportNumber", "PLTaxIdentificationNumber", "PLREGONNumber", "PTCitizenCardNumber", "PTCitizenCardNumberV2", "PTTaxIdentificationNumber", "ROPersonalNumericalCode", "RUPassportNumberDomestic", "RUPassportNumberInternational", "SANationalID", "SGNationalRegistrationIdentityCardNumber", "SKPersonalNumber", "SITaxIdentificationNumber", "SIUniqueMasterCitizenNumber", "ZAIdentificationNumber", "KRResidentRegistrationNumber", "ESDNI", "ESSocialSecurityNumber", "ESTaxIdentificationNumber", "SQLServerConnectionString", "SENationalID", "SENationalIDV2", "SEPassportNumber", "SETaxIdentificationNumber", "SWIFTCode", "CHSocialSecurityNumber", "TWNationalID", "TWPassportNumber", "TWResidentCertificate", "THPopulationIdentificationCode", "TRNationalIdentificationnumber", "UKDriversLicenseNumber", "UKElectoralRollNumber", "UKNationalHealthNumber", "UKNationalInsuranceNumber", "UKUniqueTaxpayerNumber", "USUKPassportNumber", "USBankAccountNumber", "USDriversLicenseNumber", "USIndividualTaxpayerIdentification", "USSocialSecurityNumber", "UAPassportNumberDomestic", "UAPassportNumberInternational", "Organization", "Email", "URL", "Age", "PhoneNumber", "IPAddress", "Date", "Person", "Address", "All", "Default"]
```

Comment:
some spelling changes based off of the linter:

`AzureIAASDatabaseConnectionandSQLString` -> `AzureIAASDatabaseConnectionAndSQLString`
all `Taxidentification` -> `TaxIdentification`
all `Identificationnumber` -> `IdentificationNumber`

Reply:
changed these as well